SCClassLibrary/Common/Files/File.sc

```diff
@@ -18,6 +18,11 @@ File : UnixFILE {
 		file = this.new(pathName, mode);
 		^{ function.value(file) }.protect({ file.close });
 	}
+	*read { arg pathName, mode = "r", selector = \readAllString;
```

Comment:
Used in ObjectiveC as well:

https://developer.apple.com/library/content/documentation/General/Conceptual/DevPedia-CocoaCore/Selector.html